### ISSUE TITLE
[Bug]: RStudio referencing old URLs (and old versions)

### ISSUE THREAD

**Issue by AScott-WWF** [aaronparker/evergreen#554]
### What happened?

It appears at some stage the domain serving R Studio (https://www.rstudio.com) has had a domain redirect put in place to https://posit.co/ (by the version numbers returned by Evergreen this change probably happened sometime between February 2022 and September 2023)
Evergreen therefore is providing RStudio versions from the old site url so can only assume the .json files are not affected by the domain redirect hence they are still accessible at the old URL, this is returning RStudio-2022.02.5-503 and old Url information.

It appears the new download site (https://posit.co/download/rstudio-desktop/) has a much later version (2023.09.0+463 - as of 29th September 2023)
Sorry I'm unsure if the new site data is held in a .json file. - Maybe this can be worked out from https://dailies.rstudio.com/?
But for now it appears Evergreen is serving incorrect / out-of-date info regarding RStudio, so is effectively in a broken state

### Version

2309.859

### What PowerShell edition/s are you running Evergreen on?

PowerShell Core, Windows PowerShell

### Which operating system/s are you running Evergreen on?

Windows 10+

### Have you reviewed the documentation?

- [X] Troubleshooting at: https://stealthpuppy.com/evergreen/troubleshoot/
- [X] Known issues at: https://stealthpuppy.com/evergreen/issues/

### Verbose output

```shell
Get-EvergreenApp -Name RStudio -Verbose
VERBOSE: Function path: C:\Program Files\WindowsPowerShell\Modules\Evergreen\2309.859\Apps\Get-RStudio.ps1
VERBOSE: Function exists: C:\Program Files\WindowsPowerShell\Modules\Evergreen\2309.859\Apps\Get-RStudio.ps1.
VERBOSE: Dot sourcing: C:\Program Files\WindowsPowerShell\Modules\Evergreen\2309.859\Apps\Get-RStudio.ps1.
VERBOSE: Get-FunctionResource: read application resource strings from [C:\Program Files\WindowsPowerShell\Modules\Evergreen\2309.859\Manifests\RStudio.json]
VERBOSE: Calling: Get-RStudio.
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: UserAgent: Mozilla/5.0 (Windows NT 10.0; Microsoft Windows 10.0.19045; en-GB) AppleWebKit/534.6 (KHTML, like Gecko) Chrome/7.0.500.0 Safari/534.6.
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: ContentType: application/json; charset=utf-8.
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: UseBasicParsing: True.
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: Method: Default.
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: Uri: https://dailies.rstudio.com/rstudio/prairie-trillium/index.json.
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: DisableKeepAlive: True.
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: MaximumRedirection: 2.
VERBOSE: HTTP/1.1 GET with 0-byte payload
VERBOSE: received 30531-byte response of content type application/json
VERBOSE: Content encoding: utf-8
VERBOSE: Get-FileType: found extension: [exe]
VERBOSE: Get-FileType: found extension: [zip]
VERBOSE: Get-FileType: found extension: [exe]
VERBOSE: Get-FileType: found extension: [zip]
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: UserAgent: Mozilla/5.0 (Windows NT 10.0; Microsoft Windows 10.0.19045; en-GB) AppleWebKit/534.6 (KHTML, like Gecko) Chrome/7.0.500.0 Safari/534.6.
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: ContentType: application/json; charset=utf-8.
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: UseBasicParsing: True.
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: Method: Default.
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: Uri: https://dailies.rstudio.com/rstudio/prairie-trillium/index.json.
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: DisableKeepAlive: True.
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: MaximumRedirection: 2.
VERBOSE: HTTP/1.1 GET with 0-byte payload
VERBOSE: received 30531-byte response of content type application/json
VERBOSE: Content encoding: utf-8
VERBOSE: Get-FileType: found extension: [exe]
VERBOSE: Get-FileType: found extension: [zip]
VERBOSE: Get-FileType: found extension: [exe]
VERBOSE: Get-FileType: found extension: [zip]
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: UserAgent: Mozilla/5.0 (Windows NT 10.0; Microsoft Windows 10.0.19045; en-GB) AppleWebKit/534.6 (KHTML, like Gecko) Chrome/7.0.500.0 Safari/534.6.
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: ContentType: application/json; charset=utf-8.
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: UseBasicParsing: True.
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: Method: Default.
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: Uri: https://dailies.rstudio.com/rstudio/prairie-trillium/index.json.
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: DisableKeepAlive: True.
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: MaximumRedirection: 2.
VERBOSE: HTTP/1.1 GET with 0-byte payload
VERBOSE: received 30531-byte response of content type application/json
VERBOSE: Content encoding: utf-8
VERBOSE: Get-FileType: found extension: [exe]
VERBOSE: Get-FileType: found extension: [zip]
VERBOSE: Get-FileType: found extension: [exe]
VERBOSE: Get-FileType: found extension: [zip]
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: UserAgent: Mozilla/5.0 (Windows NT 10.0; Microsoft Windows 10.0.19045; en-GB) AppleWebKit/534.6 (KHTML, like Gecko) Chrome/7.0.500.0 Safari/534.6.
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: ContentType: application/json; charset=utf-8.
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: UseBasicParsing: True.
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: Method: Default.
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: Uri: https://dailies.rstudio.com/rstudio/juliet-rose/index.json.
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: DisableKeepAlive: True.
VERBOSE: Invoke-RestMethodWrapper: Invoke-RestMethod parameter: MaximumRedirection: 2.
VERBOSE: HTTP/1.1 GET with 0-byte payload
VERBOSE: received 18093-byte response of content type application/json
VERBOSE: Content encoding: utf-8
VERBOSE: Get-FileType: found extension: [exe]
VERBOSE: Get-FileType: found extension: [zip]
VERBOSE: Get-FileType: found extension: [exe]
VERBOSE: Get-FileType: found extension: [zip]

Version       : 1.4.1743
Sha256        : 2d27888c84d434c623fbdf793e7f4364eac1acd2913efa406a94fe4ae1f46c02
Size          : 156179368
Branch        : Juliet Rose
Channel       : Preview
ProductName   : RStudio Desktop
InstallerName : Windows 10+
Type          : exe
URI           : https://s3.amazonaws.com/rstudio-ide-build/desktop/windows/RStudio-1.4.1743.exe

Version       : 1.4.1743
Sha256        : dd96cf1193108c9cd46aa450e8948aa5df7c1ae065dc45219eac3870d05fb6bc
Size          : 227780990
Branch        : Juliet Rose
Channel       : Preview
ProductName   : RStudio Desktop
InstallerName : Windows 10+ (installer-less)
Type          : zip
URI           : https://s3.amazonaws.com/rstudio-ide-build/desktop/windows/RStudio-1.4.1743.zip

Version       : 1.4.1743-4
Sha256        : d54593e712749e85b1049653b46fee416b40ca045e78a7431bae576fdbbf4f68
Size          : 161044432
Branch        : Juliet Rose
Channel       : Preview
ProductName   : RStudio Desktop Pro
InstallerName : Windows 10+
Type          : exe
URI           : https://s3.amazonaws.com/rstudio-ide-build/desktop/windows/RStudio-pro-1.4.1743-4.exe

Version       : 1.4.1743-4
Sha256        : 2d255f43b5d104062d69f30ba14b0fd2720cc79c2a7aea33a14f9fc90e45b4bd
Size          : 234551432
Branch        : Juliet Rose
Channel       : Preview
ProductName   : RStudio Desktop Pro
InstallerName : Windows 10+ (installer-less)
Type          : zip
URI           : https://s3.amazonaws.com/rstudio-ide-build/desktop/windows/RStudio-pro-1.4.1743-4.zip

Version       : 2022.02.5+503
Sha256        : 16639b001962ff13bab862a94ee28e863ebdd4cacc0a1db6c248287b9a86de3f
Size          : 177272768
Branch        : Spotted Wakerobin
Channel       : Release
ProductName   : RStudio Desktop
InstallerName : Windows 10+
Type          : exe
URI           : https://s3.amazonaws.com/rstudio-ide-build/desktop/windows/RStudio-2022.02.5-503.exe

Version       : 2022.02.5+503
Sha256        : 5adf922c9cf9c4d24022d535ccb6fea8f06ef0994ae6d5a6a2a8aef63f3625d3
Size          : 245872824
Branch        : Spotted Wakerobin
Channel       : Release
ProductName   : RStudio Desktop
InstallerName : Windows 10+ (installer-less)
Type          : zip
URI           : https://s3.amazonaws.com/rstudio-ide-build/desktop/windows/RStudio-2022.02.5-503.zip

Version       : 2022.02.5+503.pro1
Sha256        : 75240257b5fb70ca2fd4187172e4c720ebe5b2d15586afcc88c80e24cda05465
Size          : 181033968
Branch        : Spotted Wakerobin
Channel       : Release
ProductName   : RStudio Desktop Pro
InstallerName : Windows 10+
Type          : exe
URI           : https://s3.amazonaws.com/rstudio-ide-build/desktop/windows/RStudio-pro-2022.02.5-503.pro1.exe

Version       : 2022.02.5+503.pro1
Sha256        : f1204e37448d478e74924c893ec16bdae11bb36453850efdcdb97c4df62f9741
Size          : 250811754
Branch        : Spotted Wakerobin
Channel       : Release
ProductName   : RStudio Desktop Pro
InstallerName : Windows 10+ (installer-less)
Type          : zip
URI           : https://s3.amazonaws.com/rstudio-ide-build/desktop/windows/RStudio-pro-2022.02.5-503.pro1.zip

Version       : 2022.02.5+503
Sha256        : 16639b001962ff13bab862a94ee28e863ebdd4cacc0a1db6c248287b9a86de3f
Size          : 177272768
Branch        : Prairie Trillium
Channel       : Release
ProductName   : RStudio Desktop
InstallerName : Windows 10+
Type          : exe
URI           : https://s3.amazonaws.com/rstudio-ide-build/desktop/windows/RStudio-2022.02.5-503.exe

Version       : 2022.02.5+503
Sha256        : 5adf922c9cf9c4d24022d535ccb6fea8f06ef0994ae6d5a6a2a8aef63f3625d3
Size          : 245872824
Branch        : Prairie Trillium
Channel       : Release
ProductName   : RStudio Desktop
InstallerName : Windows 10+ (installer-less)
Type          : zip
URI           : https://s3.amazonaws.com/rstudio-ide-build/desktop/windows/RStudio-2022.02.5-503.zip

Version       : 2022.02.5+503.pro1
Sha256        : 75240257b5fb70ca2fd4187172e4c720ebe5b2d15586afcc88c80e24cda05465
Size          : 181033968
Branch        : Prairie Trillium
Channel       : Release
ProductName   : RStudio Desktop Pro
InstallerName : Windows 10+
Type          : exe
URI           : https://s3.amazonaws.com/rstudio-ide-build/desktop/windows/RStudio-pro-2022.02.5-503.pro1.exe

Version       : 2022.02.5+503.pro1
Sha256        : f1204e37448d478e74924c893ec16bdae11bb36453850efdcdb97c4df62f9741
Size          : 250811754
Branch        : Prairie Trillium
Channel       : Release
ProductName   : RStudio Desktop Pro
InstallerName : Windows 10+ (installer-less)
Type          : zip
URI           : https://s3.amazonaws.com/rstudio-ide-build/desktop/windows/RStudio-pro-2022.02.5-503.pro1.zip

Version       : 2022.02.5+503
Sha256        : 16639b001962ff13bab862a94ee28e863ebdd4cacc0a1db6c248287b9a86de3f
Size          : 177272768
Branch        : Ghost Orchid
Channel       : Release
ProductName   : RStudio Desktop
InstallerName : Windows 10+
Type          : exe
URI           : https://s3.amazonaws.com/rstudio-ide-build/desktop/windows/RStudio-2022.02.5-503.exe

Version       : 2022.02.5+503
Sha256        : 5adf922c9cf9c4d24022d535ccb6fea8f06ef0994ae6d5a6a2a8aef63f3625d3
Size          : 245872824
Branch        : Ghost Orchid
Channel       : Release
ProductName   : RStudio Desktop
InstallerName : Windows 10+ (installer-less)
Type          : zip
URI           : https://s3.amazonaws.com/rstudio-ide-build/desktop/windows/RStudio-2022.02.5-503.zip

Version       : 2022.02.5+503.pro1
Sha256        : 75240257b5fb70ca2fd4187172e4c720ebe5b2d15586afcc88c80e24cda05465
Size          : 181033968
Branch        : Ghost Orchid
Channel       : Release
ProductName   : RStudio Desktop Pro
InstallerName : Windows 10+
Type          : exe
URI           : https://s3.amazonaws.com/rstudio-ide-build/desktop/windows/RStudio-pro-2022.02.5-503.pro1.exe

Version       : 2022.02.5+503.pro1
Sha256        : f1204e37448d478e74924c893ec16bdae11bb36453850efdcdb97c4df62f9741
Size          : 250811754
Branch        : Ghost Orchid
Channel       : Release
ProductName   : RStudio Desktop Pro
InstallerName : Windows 10+ (installer-less)
Type          : zip
URI           : https://s3.amazonaws.com/rstudio-ide-build/desktop/windows/RStudio-pro-2022.02.5-503.pro1.zip
```

**Comment by aaronparker** [comment]
`RStudio` has been using the download feed for the dailies version of the installers. This update uses the stable builds only: https://dailies.rstudio.com/json-api/

```powershell
Version     : 2023.09.0+463
ProductName : RStudio 2023.09.0+463 - Windows 10/11 (64-bit)
Pro         : False
Date        : 2023-09-28
Size        : 212484104
Sha256      : d523c72b297edb36d045bbb9ddab5017e5ff261ffbcd2cedb0e4c5f00ce2acaa
Type        : exe
URI         : https://download1.rstudio.org/electron/windows/RStudio-2023.09.0-463.exe

Version     : 2023.09.0+463.pro11
ProductName : RStudio Pro 2023.09.0+463.pro11 - Windows 10/11 (64-bit)
Pro         : True
Date        : 2023-09-28
Size        : 216466256
Sha256      : b0331287b1605028ddec054db3f544bba276883202fe11b6afa5ad7fd640dee0
Type        : exe
URI         : https://download1.rstudio.org/electron/windows/RStudio-pro-2023.09.0-463.pro11.exe
```

**Comment by aaronparker** [comment]
This is now available in the released version `2310.868`
